cdk/cdk-resources/alert-configuration/README.md

```diff
@@ -0,0 +1,51 @@
+# mongodb-atlas-alertconfiguration-cdktf
```

Comment:
```suggestion
# @mongodb-awscdk/atlas-alertconfiguration
```
follow this format ^^ for all resources, MongoDB is the publisher with suffix of -awscdk/atlas-resourcename. see conversation in AWS CDK team slack channel